src/Microsoft.ML.Data/Data/Conversion.cs

```diff
@@ -899,6 +909,18 @@ public void Convert(in BL src, ref SB dst)
         public void Convert(in DT src, ref SB dst) { ClearDst(ref dst); dst.AppendFormat("{0:o}", src); }
         public void Convert(in DZ src, ref SB dst) { ClearDst(ref dst); dst.AppendFormat("{0:o}", src); }
         #endregion ToStringBuilder
+        #region ToBL
+        public void Convert(in R8 src, ref BL dst) => dst = src > 0 ? true : false;
```

Comment:
```suggestion
        public void Convert(in R8 src, ref BL dst) => dst = src > 0.5 ? true : false;
```

Reply:
I'm not sure about that.

---
In reply to: [263190893](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263190893) [](ancestors = 263190893)

Reply:
I think 0.00000001 looks more closer to 0?

---
In reply to: [263196830](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263196830) [](ancestors = 263196830,263190893)